src/main/java/jsr223/perl/utils/Log4jConfigurationLoader.java

```diff
@@ -25,7 +25,7 @@
  */
 package jsr223.perl.utils;
 
-public class PerlLog4jConfigurationLoader {
+public class Log4jConfigurationLoader {
```

Comment:
All the class were renamed with Perl prefix by purpose. Because during the execution in scheduler it happened to execute wrong class as the names of classes were the same as in docker-compose project.
Why do you want to rename only this class?